versioner/nonsemanticversion.go

```diff
@@ -0,0 +1,108 @@
+package versioner
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var (
+	// ErrInvalidNonSemanticVer is returned if a version is found to be invalid when
+	// being parsed.
+	ErrInvalidNonSemanticVer = fmt.Errorf("Invalid NonSemantic Version")
```

Comment:
i think we can call the basic version just 'version' instead of 'nonsemantic version'. references to the absence of semantic versioning in this package are confusing.